frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -1,63 +1,194 @@
-import React from "react";
-import { Alert, Form, Row, Col, Input, Button, Select, Collapse, Tooltip, Checkbox } from "antd";
+import React, { useState } from "react";
+import {
+  Alert,
+  Form,
+  Row,
+  Col,
+  Input,
+  Button,
+  Select,
+  Collapse,
+  Tooltip,
+  Checkbox,
+  type FormInstance,
+} from "antd";
 import { useSelector } from "react-redux";
-import type { OxalisState, UserBoundingBox } from "oxalis/store";
-import { getUserBoundingBoxesFromState } from "oxalis/model/accessors/tracing_accessor";
+import type { HybridTracing, OxalisState, UserBoundingBox, VolumeTracing } from "oxalis/store";
+import { getSomeTracing } from "oxalis/model/accessors/tracing_accessor";
 import {
   getColorLayers,
   getResolutionInfo,
-  getSegmentationLayerByName,
   getSegmentationLayers,
 } from "oxalis/model/accessors/dataset_accessor";
-import { runTraining } from "admin/admin_rest_api";
+import {
+  getAnnotationInformation,
+  getDataset,
+  getTracingForAnnotationType,
+  runTraining,
+} from "admin/admin_rest_api";
 import { LayerSelection, LayerSelectionFormItem } from "components/layer_selection";
 import Toast from "libs/toast";
 import { Model } from "oxalis/singletons";
-import { getReadableNameForLayerName } from "oxalis/model/accessors/volumetracing_accessor";
+import { getSegmentationLayerByHumanReadableName } from "oxalis/model/accessors/volumetracing_accessor";
 import _ from "lodash";
 import BoundingBox from "oxalis/model/bucket_data_handling/bounding_box";
 import { formatVoxels } from "libs/format_utils";
+import * as Utils from "libs/utils";
 import { V3 } from "libs/mjs";
+import type { APIAnnotation, APIDataset, ServerVolumeTracing } from "types/api_flow_types";
+import type { Vector3 } from "oxalis/constants";
+import { serverVolumeToClientVolumeTracing } from "oxalis/model/reducers/volumetracing_reducer";
+import { convertUserBoundingBoxesFromServerToFrontend } from "oxalis/model/reducers/reducer_helpers";
 
 const { TextArea } = Input;
 const FormItem = Form.Item;
 
+export type AnnotationInfoForAIJob<GenericAnnotation> = {
```

Comment:
Please note that I also added this comment. It hopefully explains why all these additional properties (`volumeTracings, userBoundingBoxes, volumeTracingResolutions`) are needed. Please tell me your opinion on whether you understand what I want to express and maybe on wording improvements :pray: